app/tests/cases_tests/test_tiff.py

```diff
@@ -146,6 +159,8 @@ def test_tiff_image_entry_creation(resource, expected_error_message):
         assert image_entry.color_space == get_color_space(
             str(tiff_tags["PhotometricInterpretation"].value)
         )
+        assert image_entry.pixel_height == pixel_height
+        assert image_entry.pixel_width == pixel_width
```

Comment:
Probably best to use https://docs.pytest.org/en/latest/reference.html#pytest-approx here